testsuite/MDAnalysisTests/lib/test_util.py

```diff
@@ -21,6 +21,8 @@
 #
 from __future__ import absolute_import, division
 
+from unittest import TestCase
```

Comment:
Is using `unittest` really necessary? I thought we can get rid of it. `pytest` supports `setup_method()/teardown_method()` (instead of `setUp()`) https://docs.pytest.org/en/2.7.3/xunit_setup.html

(I know that it also supports running `UnitTest` classes.)